ols/app/models/config.py

```diff
@@ -314,6 +314,7 @@ class PostgresConfig(BaseModel):
     password_path: Optional[str] = None
     password: Optional[str] = None
     require_ssl: bool = False
+    ssl_mode: str = constants.POSTGRES_CACHE_SSL_MODE
```

Comment:
"require_ssl" and "ssl_mode" seem redundant, isn't one of the possible ssl modes "require"?

Reply:
yup, it was leftover, updated now